### PR TITLE
Specify to use staging branch for development

### PR DESCRIPTION
**Pull request by fureigh**
Following the lead of [18f.gsa.gov's README](https://github.com/18F/18f.gsa.gov/blob/staging/README.md).

### CODE REVIEW

**Comment by maya** [comment]
Should we specify that the staging branch is `18f-pages-staging`?

**Comment by fureigh** [comment]
@maya Absolutely. Good catch.

File: README.md [hunk 1]
```diff
@@ -67,6 +67,17 @@ Questions or need help with setup? Feel free to open an issue here [https://gith
 
 See [CONTRIBUTING](CONTRIBUTING.md).
 
+### Deployment and workflow
+
+* The staging branch `18f-pages-staging` is **automatically deployed** to our staging site: https://pages-staging.18f.gov/designstandards/.
+* The production branch `18f-pages` is **automatically deployed** to our production site: https://pages.18f.gov/designstandards/.
```
Comment (maya) [inline README.md]: For our production site, we currently have two: https://pages.18f.gov/designstandards/ and https://playbook.cio.gov/designstandards/. Should we state both? 

18F pages (https://pages.18f.gov/designstandards/) is the faster, more reliable one, but https://playbook.cio.gov/designstandards/ is what is publically known.

Comment (maya) [inline README.md]: Updated to the publicly known one here: 6d3eb63

**Pull request merged**